src/parameters.rs

```diff
@@ -32,6 +33,36 @@ pub struct MetaCallArgs {
     pub args: Vec<u8>,
 }
 
+/// Borsch-encoded log for use in a `RawCallResult`.
+#[derive(Debug, BorshSerialize, BorshDeserialize)]
+pub struct ResultLog {
+    pub topics: Vec<RawU256>,
+    pub data: Vec<u8>,
+}
+
+impl From<Log> for ResultLog {
+    fn from(log: Log) -> Self {
+        let mut res_topics = Vec::with_capacity(log.topics.len());
```

Comment:
Nit: this transformation could also be done with `map` + `collect` instead of with a mutable Vec and for loop.

I generally think the `map` version is easier to read and fewer lines of code. But it's not a PR blocker by any stretch.

Reply:
No idea how I keep missing these hah! Map + collect is one of my fav features of Rust.

Reply:
Ok, this is resolved.

Reply:
Sorted